### PR TITLE
FIX mne setup_source_spaces version conflict

### PR DESCRIPTION
**Pull request by kambysese**
@Eric89GXL filename arg is deprecated and causing issues in `mne.setup_source_space`

### CODE REVIEW

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -1589,12 +1590,14 @@ def gen_forwards(p, subjects, structurals, run_indices):
...
-            src = op.join(subjects_dir, structurals[si], 'bem',
-                          structurals[si] + '-oct-6-src.fif')
-            if not op.isfile(src):
+            src_space_file = op.join(subjects_dir, structurals[si], 'bem',
+                                     structurals[si] + '-oct-6-src.fif')
+            if not op.isfile(src_space_file):
                 print('  Creating source space for %s...' % subj)
-                setup_source_space(structurals[si], src, 'oct6',
-                                   n_jobs=p.n_jobs)
+                src = setup_source_space(structurals[si], 'oct6',
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: better to pass `spacing='oct6'` to help the transition

Comment (kambysese) [inline mnefun/_mnefun.py]: Will do. I was actually thinking of making spacing a Params arg, WDYT?

Comment (Eric89GXL) [inline mnefun/_mnefun.py]: Do you need to change it from the recommended default?

Comment (kambysese) [inline mnefun/_mnefun.py]: Not me. But I also never thought I'd be doing MEG in populations from the recommended default. I'll leave it as is.

Comment (Eric89GXL) [inline mnefun/_mnefun.py]: YAGNI

Comment (kambysese) [inline mnefun/_mnefun.py]: Done

**Comment by Eric89GXL** [comment]
Thanks @kambysese

**Pull request merged**